buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -340,30 +340,31 @@ class BeamModulePlugin implements Plugin<Project> {
     // These versions are defined here because they represent
     // a dependency version which should match across multiple
     // Maven artifacts.
+    def apex_core_version = "3.7.0"
```

Comment:
I see a lot of changes of version variables below. Is it caused by just reordering them? Did you keep the same version values (except `aws_java_sdk_version`)?

Reply:
yes exactly it is just a reorder to make things easier to find.